src/Controls/src/Core/Compatibility/Handlers/ListView/iOS/EntryCellRenderer.cs

```diff
@@ -28,23 +28,23 @@ public override UITableViewCell GetCell(Cell item, UITableViewCell reusableCell,
 
 			var tvc = reusableCell as EntryCellTableViewCell;
 			if (tvc == null)
+			{
 				tvc = new EntryCellTableViewCell(item.GetType().FullName);
+			}
 			else
 			{
-				tvc.PropertyChanged -= HandlePropertyChanged;
+				CellPropertyChanged -= HandlePropertyChanged;
```

Comment:
Because this even now is just local to this class this isn't going to cause a circular reference. I opted to keep this event path because this maintains the timing of when `HandlePropertyChanged` will start and stop firing